src/note.ts

```diff
@@ -387,10 +387,8 @@ export abstract class Note extends Tickable {
   }
 
   /** Gets the stave line number for the note. */
-  getLineNumber(
-    // eslint-disable-next-line
-    isTopNote?: boolean
-  ): number {
+  // eslint-disable-next-line
+  getLineNumber(isTopNote?: boolean): number {
```

Comment:
Have you checked if typedoc copes with the comment? Is the function still documented? (use `grunt typedoc` and have a look into build/docs)

Reply:
Thanks for your review. I haven't checked typedoc yet, and I will do so to make sure that the eslint-disable-next-line comments don't break the typedoc.

The only reason I've been moving the eslint-disable-next-line up to the line before the method is due to aesthetics. I definitely prefer if we don't have a comment in the middle of the method signature.

However, if it breaks typedoc, then I'll have to live with it! (So far I have been depending on the mouseover feature in VS code to check that the documentation looks OK. I just move my mouse cursor over the method name and a tooltip shows the comment.)

Reply:
It looks like typedoc generates the correct documentation even if the code has the eslint-disable-next-line right before the method:

```
  /** Gets the stave line number for the note. */
  // eslint-disable-next-line
  getLineNumber(isTopNote?: boolean): number {
    return 0;
  }
```

@rvilarl is the above OK with you? Otherwise, the eslint directive breaks up the method signature and makes the code a bit harder to read, in my opinion.

Reply:
This is fine, I made it because I thought that it would break typedoc. It looks much better this way.